client.go

```diff
@@ -330,6 +336,7 @@ func (c *ovndb) reconnect() {
 func (c *ovndb) MonitorTables(jsonContext interface{}) (*libovsdb.TableUpdates, error) {
 	// get the table list based on the DB
 	var tables []string
+	c.AddLatestDBSchemaTables()
```

Comment:
L339 is inserted in-between L338-344 and the comment @L337 was explaining that. Perhaps you need to move this line to L337

Reply:
okay, will do the change.